src/plasoscaffolder/dal/sqlite_query_execution.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=no-member
+# pylint does not recognize connect and close as member
+"""Base for sql query validators"""
+import sqlite3
+
+from plasoscaffolder.dal import base_sql_query_execution
+
+
+class SQLQueryExecution(base_sql_query_execution.BaseSQLQueryExecution):
+  """Class representing the SQLite query validator
+  """
+
+  def __init__(self, database_path: str):
+    """Initializes the SQL Query Validator
+
+    Args:
+      database_path: the path to the sqlite database schema
+    """
+    self._connection = sqlite3.connect(database_path)
```

Comment:
Design considerations:
* what if the data base file cannot be opened? restricted access, unreadable, corrupt?